src/cargo/core/features.rs

```diff
@@ -657,6 +657,7 @@ unstable_cli_options!(
     // See: https://github.com/rust-lang/cargo/pull/9525#discussion_r728470927
     rustdoc_scrape_examples: Option<String> = ("Allow rustdoc to scrape examples from reverse-dependencies for documentation"),
     skip_rustdoc_fingerprint: bool = (HIDDEN),
+    extra_link_arg_etc: bool = ("Allow specifying link arg for tests,examples and benches"),
```

Comment:
```suggestion
    extra_link_arg_etc: bool = ("Allow specifying link arg for tests, examples, and benches"),
```

Reply:
Doesn't matter as @ehuss is okay with no flags. Thanks anyway! :)